src/gigs-board/entity/post/PostEditor.jsx

```diff
@@ -142,7 +139,12 @@ const onSubmit = () => {
     },
     Submission: {
       name: state.name,
-      description: state.description,
+      description: generateDescription(
+        state.description,
+        state.amount,
+        state.token,
+        state.supervisor
+      ),
```

Comment:
If really to remove fields from sponsorship, below Sponsorship:
```
 Sponsorship: {
      name: state.name,
      description: state.description,
      amount: state.amount,
      sponsorship_token: state.token,
      supervisor: state.supervisor,
      sponsorship_version: "V1",
    },
```

 should also update, and call contract to create sponsorship should also update. I suggest to consider & address sponsorship in a future PR. in this PR, only do fix #142.